src/diffusers/modular_pipelines/qwenimage/decoders.py

```diff
@@ -74,8 +74,9 @@ def __call__(self, components: QwenImageModularPipeline, state: PipelineState) -
         block_state = self.get_block_state(state)
 
         # YiYi Notes: remove support for output_type = "latents', we can just skip decode/encode step in modular
+        vae_scale_factor = 2 ** len(components.vae.temperal_downsample)
```

Comment:
Keeping `vae_scale_factor` fixed to 8, for example, would break the tests as we use a smaller VAE.